app/models/pydantic/jobs.py

```diff
@@ -27,10 +27,10 @@ class Job(StrictBaseModel):
     job_definition: str
     command: List[str]
     environment: List[Dict[str, str]] = []
-    vcpus: int
-    memory: int
-    attempts: int
-    attempt_duration_seconds: int
+    vcpus: int = 8
+    memory: int = 3200
+    attempts: int = 1
+    attempt_duration_seconds: int = 7200
```

Comment:
I think we specify these in all the child classes so it shouldn't matter, but curious why add defaults?